src/gitmoji.js

```diff
@@ -75,6 +75,24 @@ class GitmojiCli {
 		};
 	}
 
+	remove() {
+		const hookFile = 'prepare-commit-msg';
```

Comment:
Can you delete `hookFile` and `path` constants from `remove()` and init `methods()` and create a global `commitHookPath` constant outside the class. Code duplication is dangerous! ⚠️